CMakeLists.txt

```diff
@@ -643,6 +639,12 @@ if (WIN32 OR is_xcode OR unity)
 
   list(APPEND src ${beast_unity_srcs} ${ripple_unity_srcs})
 
+  add_with_props(src/unity/test_unity.cpp
```

Comment:
@bachase Yes, that's the right function to add a source that add properties just for that source.